pkgs/tools/system/osquery/default.nix

```diff
@@ -0,0 +1,82 @@
+{ pkgs }:
+
+with pkgs;
```

Comment:
you should be explicit about your dependencies like other packaged software
and include them in your whole signature function, they will be provided directly

Reply:
```suggestion
```